### PR TITLE
Fix a typo and delete a repeated word

### PR DESCRIPTION
**Pull request by netyster**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1112/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/5 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/6 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/7 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/8 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/9 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/10 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/11 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/12 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/13 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/14 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/15 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/16 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/17 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/18 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/19 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by astonzhang** [comment]
@bwerness could you take a look?

**Comment by mli** [comment]
Job d2l-en/PR-1112/20 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/21 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/22 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Job d2l-en/PR-1112/23 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by mli** [comment]
Thanks for fixing these issues. Is there PR ready to review or @netyster you are still working on it?

File: chapter_appendix-mathematics-for-deep-learning/information-theory.md [hunk 1]
```diff
@@ -409,7 +409,7 @@ q1 = torch.sort(q1)[0]
 q2 = torch.sort(q2)[0]
 ```
 
-Since $q_1$ and $q_2$ are symmetric with respect to the y-axis (i.e., $x=0$), we expect a similar value of KL divergence between $D_{\mathrm{KL}}(p\|q_1)$ and $D_{\mathrm{KL}}(p\|q_2)$. As you can see below, there is only a 1% off between $D_{\mathrm{KL}}(p\|q_1)$ and $D_{\mathrm{KL}}(p\|q_2)$.
+Since $q_1$ and $q_2$ are symmetric with respect to the y-axis (i.e., $x=0$), we expect a similar value of KL divergence between $D_{\mathrm{KL}}(p\|q_1)$ and $D_{\mathrm{KL}}(p\|q_2)$. As you can see below, there is only a 3% off between $D_{\mathrm{KL}}(p\|q_1)$ and $D_{\mathrm{KL}}(p\|q_2)$.
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/information-theory.md:412]: This number is a bit tricky. Can you change to "a less than 3% off"? Thanks!

File: chapter_appendix-mathematics-for-deep-learning/information-theory.md [hunk 2]
```diff
@@ -428,7 +428,7 @@ similar_percentage = abs(kl_pq1 - kl_pq2) / ((kl_pq1 + kl_pq2) / 2) * 100
 kl_pq1, kl_pq2, similar_percentage
 ```
 
-In contrast, you may find that $D_{\mathrm{KL}}(q_2 \|p)$ and $D_{\mathrm{KL}}(p \| q_2)$ are off a lot, with around 40% off as shown below.
+In contrast, you may find that $D_{\mathrm{KL}}(q_2 \|p)$ and $D_{\mathrm{KL}}(p \| q_2)$ are off a lot, with around 46% off as shown below.
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/information-theory.md:431]: This is a rough number "around 40%". (Every time the seed gives us different results).

File: chapter_appendix-mathematics-for-deep-learning/integral-calculus.md [hunk 3]
```diff
@@ -109,10 +109,10 @@ x = torch.arange(a, b, epsilon)
 f = x / (1 + x**2)
 
 approx = torch.sum(epsilon*f)
-true = torch.log(torch.tensor([2.])) / 2
+true = torch.log(torch.tensor([5.])) / 2
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/integral-calculus.md:112]: Can you specify why changing to a "5" here?

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md [hunk 4]
```diff
@@ -447,11 +447,11 @@ $$
 \begin{aligned}
 f(0,0) & = a, \\
 \nabla f (0,0) & = \begin{bmatrix}b_1 \\ b_2\end{bmatrix}, \\
-\mathbf{H} f (0,0) & = \begin{bmatrix}2 c_{11} & c_{12} \\ c_{12} & 2c_{22}\end{bmatrix}.
+\mathbf{H} f (0,0) & = \begin{bmatrix}2 c_{11} & c_{12} \\ c_{12} & 2c_{22}\end{bmatrix},
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:450]: Here is the end of formula, so we use period.

Comment (goldmermaid) [reply chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:450]: Can you change back? Thanks!

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md [hunk 5]
```diff
@@ -576,13 +576,13 @@ $$
...
 \beta_n
-\end{bmatrix} = \boldsymbol{\beta}^\top.
+\end{bmatrix} = \boldsymbol{\beta}.
 $$
 
 This illustrates a few factors about matrix calculus that we will often counter throughout this section:
 
 * First, The computations will get rather involved.
-* Second, The final results are much cleaner than the intermediate process, and will always look similar to the single variable case.  In this case, note that $\frac{d}{dx}(bx) = b$ and $\frac{d}{d\mathbf{x}} (\boldsymbol{\beta}\mathbf{x}) = \boldsymbol{\beta}^\top$ are both similar. 
+* Second, The final results are much cleaner than the intermediate process, and will always look similar to the single variable case.  In this case, note that $\frac{d}{dx}(bx) = b$ and $\frac{d}{d\mathbf{x}} (\boldsymbol{\beta}^\top\mathbf{x}) = \boldsymbol{\beta}$ are both similar.
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:585]: I know it might be tricky here that the \beta is a **row** vector, rather than conventionally a **column** vector. 

I like this updated version. So can you add one more modification:
By changing "Suppose that we have some fixed row vector  𝜷" to "Suppose that we have some fixed column vector  𝜷"?

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md [hunk 6]
```diff
@@ -750,7 +750,7 @@ It is reasonable to ask at this point, "Why can I not just write down matrix ver
 * Matrix calculus allows us to write the derivatives of matrix expressions in concise ways.
 
 ## Exercises
-1. Given a row vector $\boldsymbol{\beta}$, compute the derivatives of both $f(\mathbf{x}) = \boldsymbol{\beta}\mathbf{x}$ and $g(\mathbf{x}) = \mathbf{x}^\top\boldsymbol{\beta}^\top$.  Why do you get the same answer?
+1. Given a row vector $\boldsymbol{\beta}^\top$, compute the derivatives of both $f(\mathbf{x}) = \boldsymbol{\beta}^\top\mathbf{x}$ and $g(\mathbf{x}) = \mathbf{x}^\top\boldsymbol{\beta}$.  Why do you get the same answer?
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:753]: Can you change it to column vector as well?

File: chapter_appendix-mathematics-for-deep-learning/statistics.md [hunk 7]
```diff
@@ -127,17 +127,17 @@ It is intuitively clear that these two components contribute to the mean squared
 $$
 \begin{aligned}
 \mathrm{MSE} (\hat{\theta}_n, \theta) &= E[(\hat{\theta}_n - E(\hat{\theta}_n) + E(\hat{\theta}_n) - \theta)^2] \\
- &= E[(\hat{\theta}_n - E(\hat{\theta}_n))^2] + E[(E(\hat{\theta}_n) - \theta)^2] \\
+ &\approx E[(\hat{\theta}_n - E(\hat{\theta}_n))^2] + E[(E(\hat{\theta}_n) - \theta)^2] \\
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/statistics.md:130]: This is an exact equal, since the "omitted" term is zero.

File: chapter_appendix-mathematics-for-deep-learning/statistics.md [hunk 8]
```diff
@@ -229,9 +229,9 @@ To complete the story of how to work with hypothesis testing, we need to now int
 
 The *statistical significance* measures the probability of erroneously rejecting the null hypothesis, $H_0$, when it should not be rejected, i.e.,
 
-$$ \text{statistical significance }= 1 - \alpha = P(\text{reject } H_0 \mid H_0 \text{ is true} ).$$
+$$ \text{statistical significance }= 1 - \alpha = 1 - P(\text{reject } H_0 \mid H_0 \text{ is true} ).$$
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/statistics.md:232]: This is the definition of statistical significance. I don't think it is a mistake. Can you change it back?

File: chapter_appendix-mathematics-for-deep-learning/statistics.md [hunk 9]
```diff
@@ -243,7 +243,7 @@ It is also referred to as the *type I error* or *false positive*. The $\alpha$,
 
 The *statistical power* (or *sensitivity*) measures the probability of reject the null hypothesis, $H_0$, when it should be rejected, i.e.,
 
-$$ \text{statistical power }= P(\text{reject } H_0  \mid H_0 \text{ is false} ).$$
+$$ \text{statistical power }= 1 - \beta = P(\text{ fail to reject } H_0  \mid H_0 \text{ is false} ).$$
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/statistics.md:246]: This is the definition of statistical power... Can you change it back?

File: chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md [hunk 10]
```diff
@@ -414,7 +412,7 @@ d2l.plot(torch.arange(1, 100), norm_ratio_list, 'Iteration', 'Ratio')
 ```
 
 If we look at the last portion of the above computation, 
-we see that the random vector is stretched by a factor of `1.974459321485[...]`,
+we see that the random vector is stretched by a factor of `2.453232963362498[...]`,
```
[the remark this answers is not in the document]
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md:415]: This is the result of Pytorch code. I have just noticed the 1.9744 is from MXNET version.
Need I change it back?

File: chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md [hunk 11]
```diff
@@ -210,7 +208,7 @@ $$
 \end{bmatrix},
 $$
 
-has only a single eigenvector, namely $(0, 1)$. 
+has only a single eigenvector, namely $(1, 0)$.
```
[the remark this answers is not in the document]
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md:211]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md [hunk 12]
```diff
@@ -870,7 +870,7 @@ For instance if we want to compute
 what would be traditionally written as
 
 $$
-c_{kl} = \sum_{ij} \mathbf{B}_{ijk}\mathbf{A}_{il}v_j.
+c_{kl} = \sum_{ij} \mathbf{b}_{ijk}\mathbf{a}_{il}v_j.
```
[the remark this answers is not in the document]
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md:873]: This modify comes from 18.1.31 where "a" is for element of martrix with index, while "A" is for matrix.

File: chapter_appendix-mathematics-for-deep-learning/information-theory.md (hunk 1 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/information-theory.md:412]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/information-theory.md (hunk 2 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/information-theory.md:431]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/integral-calculus.md (hunk 3 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/integral-calculus.md:112]: The integral of f = x / (1 + x**2) is log(1+x**2)/2, so it's 5 inside the log with x = 2.

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md (hunk 4 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:450]: The sentence of "If we look..." before these formulas is not finish, so I changed period and modify the sentence below which miss a verb to form a complete sentence.

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md (hunk 5 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:585]: I have changed it in commit 6ef2874021856ec2cbfe2cecc6c14a4a619a0297.
https://github.com/d2l-ai/d2l-en/pull/1112/commits/6ef2874021856ec2cbfe2cecc6c14a4a619a0297

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md (hunk 6 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:753]: The transpose of x in g(x) is a row vector, so it need the column one, which is the beta without transpose, to do dot product. 
Do I get your point?

File: chapter_appendix-mathematics-for-deep-learning/statistics.md (hunk 7 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/statistics.md:130]: The omitted term is the irreducible error.
https://en.wikipedia.org/wiki/Bias%E2%80%93variance_tradeoff
>The bias–variance decomposition is a way of analyzing a learning algorithm's expected generalization error with respect to a particular problem as a sum of three terms, the bias, variance, and a quantity called the irreducible error, resulting from noise in the problem itself.

File: chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md [hunk 13]
```diff
@@ -196,7 +194,7 @@ as the number of non-zero eigenvalues of $\mathbf{A}$.
 
 The examples could continue, but hopefully the point is clear:
 eigendecompositions can simplify many linear-algebraic computations
```
[the remark this answers is not in the document]
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md (hunk 5 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:585]: I have changed it in 6ef2874021856ec2cbfe2cecc6c14a4a619a0297.

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md (hunk 6 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:753]: The transpose of x in g(x) is a row vector which need a column one, the beta without transpose, to do dot product, so I get rid of the transpose of beta in g(x).
Do I get the point?

File: chapter_appendix-mathematics-for-deep-learning/statistics.md (hunk 8 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/statistics.md:232]: P(\text{reject } H_0 \mid H_0 \text{ is true} is eaqual to the alpha.
https://en.wikipedia.org/wiki/Statistical_significance
>More precisely, a study's defined significance level, denoted by {\displaystyle \alpha }\alpha , is the probability of the study rejecting the null hypothesis, given that the null hypothesis was assumed to be true;

File: chapter_appendix-mathematics-for-deep-learning/statistics.md (hunk 9 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/statistics.md:246]: What I intend here is to make the style of this formula be consist with the one of the statistical significance. But I make a mistake here that I miss "1-" in front of P. The orignal formula is ok. 
Could I add the missing "1-" or change it back ?

File: chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (goldmermaid) [reply chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md:415]: I see! Then we may need to fix some similar seeds for all the 3 framework. Can you change it back for now?

File: chapter_appendix-mathematics-for-deep-learning/statistics.md (hunk 9 above)
Comment (goldmermaid) [reply chapter_appendix-mathematics-for-deep-learning/statistics.md:246]: I see. Yes, let's add that " 1 - \beta = "!

File: chapter_appendix-mathematics-for-deep-learning/statistics.md (hunk 7 above)
Comment (goldmermaid) [reply chapter_appendix-mathematics-for-deep-learning/statistics.md:130]: You are right, it makes sense to add an irreducible error. While it might not be appropariate to add an "\approx" here (or it might confuse some of the readers how to derive the formula). 

I find an another proof, which explained the irreducible error is coming from the "\Var{\theta}".  Could you help correct the proof as how it proves in https://people.eecs.berkeley.edu/~jrs/189s19/lec/12.pdf ?

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md (hunk 6 above)
Comment (goldmermaid) [reply chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:753]: Both x and beta are column vectors. So yes your modification is correct, while can you change the "Given a row vector ..." to "Given a column vector..."?

File: chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/eigendecomposition.md:415]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/statistics.md (hunk 9 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/statistics.md:246]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md (hunk 4 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:450]: I have changed it in ca9e07575b26e25e4d48b79b99ed55837fe307f6.

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md (hunk 6 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/statistics.md (hunk 7 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/statistics.md]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md [hunk 14]
```diff
@@ -533,7 +533,7 @@ This forms the basis for Newton's Algorithm discussed in :numref:`sec_gd`, where
 ## A Little Matrix Calculus
 Derivatives of functions involving matrices turn out to be particularly nice.  This section can become notationally heavy, so may be skipped in a first reading, but it is useful to know how derivatives of functions involving common matrix operations are often much cleaner than one might initially anticipate, particularly given how central matrix operations are to deep learning applications.
 
-Let us begin with an example.  Suppose that we have some fixed row vector $\boldsymbol{\beta}$, and we want to take the product function $f(\mathbf{x}) = \boldsymbol{\beta}\mathbf{x}$, and understand how the dot product changes when we change $\mathbf{x}$.  
+Let us begin with an example.  Suppose that we have some fixed row vector $\boldsymbol{\beta}^\top$, and we want to take the product function $f(\mathbf{x}) = \boldsymbol{\beta}^\top\mathbf{x}$, and understand how the dot product changes when we change $\mathbf{x}$.
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:536]: Can you change the "some fixed row vector $\boldsymbol{\beta}^\top$" to "some fixed column vector $\boldsymbol{\beta}$" as well? Thanks!

File: chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md [hunk 15]
```diff
@@ -60,13 +60,13 @@ as discovering how to separate two distinct clusters of points.
...
-as the location $2$ units to the right and $3$ units up from the origin,
+Not only can we think of the vector $\mathbf{v} = [3,2]^\top$ 
+as the location $3$ units to the right and $2$ units up from the origin,
 we can also think of it as the direction itself 
-to take $2$ steps to the right and $3$ steps up. 
+to take $3$ steps to the right and $2$ steps up. 
 In this way, we consider all the vectors in figure :numref:`fig_arrow` the same.
 
-![Any vector can be visualized as an arrow in the plane.  In this case, every vector drawn is a representation of the vector $(2,3)$.](../img/ParVec.svg)
+![Any vector can be visualized as an arrow in the plane.  In this case, every vector drawn is a representation of the vector $(3,2)$.](../img/ParVec.svg)
```
Comment (goldmermaid) [inline chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md:69]: Could you change it to "[3,2]^\top"? Thanks!

Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md (hunk 14 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md (hunk 15 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md]: 47a0df3ec7b1d08a2567155328a885c413ee2dea
The (2,3) in fig. 18.1.2 should be (3,2), but I don't know how to modify the figure. Do you know?

Comment (goldmermaid) [reply chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md]: No worries. I will handle the plot.

Comment (goldmermaid) [reply chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md]: Please let me know when you are done with all the modifications. Thanks for your contributions! @netyster :)

Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md]: @goldmermaid All the modification is done.:)

**Comment by mli** [comment]
Job d2l-en/PR-1112/34 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1112/

**Comment by astonzhang** [comment]
@goldmermaid how does this PR look to you?

**Comment by astonzhang** [comment]
@bwerness feel free to add your review to this PR to the math chap. Thanks!

**Comment by goldmermaid** [comment]
All look great to me! @astonzhang

File: chapter_appendix-mathematics-for-deep-learning/distributions.md [hunk 16]
```diff
@@ -87,7 +87,7 @@ We can sample an array of arbitrary shape from a Bernoulli random variable as fo
 
 ## Discrete Uniform
 
-The next commonly encountered random variable encountered is a discrete uniform.  For our discussion here, we will assume that it is supported on the integers $\{1, 2, \ldots, n\}$, however any other set of values can be freely chosen.  The meaning of the word *uniform* in this context is that every possible value is equally likely.  The probability for each value $i \in \{1, 2, 3, \ldots, n\}$ is $p_i = \frac{1}{n}$.  We will denote a random variable $X$ with this distribution as
+The next commonly random variable encountered is a discrete uniform.  For our discussion here, we will assume that it is supported on the integers $\{1, 2, \ldots, n\}$, however any other set of values can be freely chosen.  The meaning of the word *uniform* in this context is that every possible value is equally likely.  The probability for each value $i \in \{1, 2, 3, \ldots, n\}$ is $p_i = \frac{1}{n}$.  We will denote a random variable $X$ with this distribution as
```
Comment (bwerness) [inline chapter_appendix-mathematics-for-deep-learning/distributions.md:90]: The phrase "the next commonly random variable encountered" is not grammatical.  I'd rephrase to "the next commonly encountered random variable is a discrete uniform.

File: chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md [hunk 17]
```diff
@@ -80,12 +80,12 @@ and then follow the directions given by the other, as is seen in :numref:`fig_ad
...
 By considering the identity that $\mathbf{u} = \mathbf{v} + (\mathbf{u}-\mathbf{v})$,
 we see that the vector $\mathbf{u}-\mathbf{v}$ is the direction 
-that takes us from the point $\mathbf{u}$ to the point $\mathbf{v}$.
+that takes us from the point $\mathbf{v}$ to the point $\mathbf{u}$.
 
 
 ## Dot Products and Angles
 As we saw in :numref:`sec_linear-algebra`, 
-if we take two column vectors say $\mathbf{u}$ and $\mathbf{v}$,
+if we take two column vectors saying $\mathbf{u}$ and $\mathbf{v}$,
```
Comment (bwerness) [inline chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md:88]: This change is also not grammatical. Delete the word "say" as it was superfluous.  So it should read "if we take two column vectors $\mathbf{u}$ and $\mathbf{v}$,"

File: chapter_appendix-mathematics-for-deep-learning/integral-calculus.md [hunk 18]
```diff
@@ -341,7 +341,7 @@ $$
 \int_{[a, b]\times[c, d]} f(x, y)\;dx\;dy.
 $$
 
-Suppose that we wish to compute this integral.  My claim is that we can do this by iteratively computing first the integral in say $x$ and then shifting to the integral in $y$, that is to say
+Suppose that we wish to compute this integral.  My claim is that we can do this by iteratively computing first the integral in saying $x$ and then shifting to the integral in $y$, that is to say
```
Comment (bwerness) [inline chapter_appendix-mathematics-for-deep-learning/integral-calculus.md:344]: Same as before, removing the word "say" is a better way to correct this writing tic of mine ;).

File: chapter_appendix-mathematics-for-deep-learning/distributions.md (hunk 16 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/distributions.md]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md (hunk 17 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/geometry-linear-algebraic-ops.md]: Ok!

File: chapter_appendix-mathematics-for-deep-learning/integral-calculus.md (hunk 18 above)
Comment (netyster) [reply chapter_appendix-mathematics-for-deep-learning/integral-calculus.md]: Ok!
